### PR TITLE
Base improvements to README, Makefile, etc

### PR DESCRIPTION
**Pull request by skylarmb**
One PR of many, split out for easier reviewing. The overall diff can be seen here: https://github.com/plaid/quickstart/pull/167 

- clean up Makefile commands
- clean up README
- formats `index.html` using prettier
- fixes the `.env` situation

@phoenixy1 would love your feedback on the readme: https://github.com/plaid/quickstart/tree/sb-quickstart-base-improvements

@raxod502-plaid would love your feedback on the .env / docker / makefile changes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,24 +4,26 @@ ifneq ("$(wildcard docker-compose.local.yml)","")
...
+language := node
+SUCCESS_MESSAGE := "✅ $(language) quickstart is running on http://localhost:8000"
 
 .PHONY: up
 up:
 	$(DOCKER_COMPOSE) \
 		$(DOCKER_COMPOSE_YML) \
-		$@ --detach --remove-orphans \
-		$(QUICKSTART)
+		$@ --build --detach --remove-orphans \
```
Comment (skylarmb) [inline Makefile:14]: i added `--build` to the up command to simplify things.

File: Makefile [hunk 2]
```diff
@@ -4,24 +4,26 @@ ifneq ("$(wildcard docker-compose.local.yml)","")
...
-		$(QUICKSTART)
+		$(language)
 
 .PHONY: stop build
 stop build:
 	$(DOCKER_COMPOSE) \
 		$(DOCKER_COMPOSE_YML) \
 		$@ \
-		$(QUICKSTART)
+		$(language)
```
Comment (skylarmb) [inline Makefile:29]: using lowercase args seems like convention `make up language=node`

Comment (raxod502-plaid) [reply Makefile:29]: I wouldn't necessarily say it's standard in Makefiles, but it's definitely easier to type, so I'd say it's a win for this project. You could even go with `lang=node` to be shorter, which has a tradeoff of being a little less clear.

**Review by raxod502-plaid — DISMISSED** [review]
Seems like a great improvement!

**Review by Mendoza920 — CHANGES_REQUESTED** [review]
What do I do

**Review by Mendoza920 — APPROVED** [review]

**Comment by Mendoza920** [comment]
I need an exorcism

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Get your Plaid API keys from the dashboard: https://dashboard.plaid.com/account/keys
+PLAID_CLIENT_ID=
```
Comment (Mendoza920) [inline .env.example:2]: 5fac2a3d21746b0017dd8c28

File: .env.example [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+PLAID_SECRET=
```
Comment (Mendoza920) [inline .env.example:3]: I am not sure how to do this.

File: .env.example [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# Use 'sandbox' to test with fake credentials in Plaid's Sandbox environment
```
Comment (Mendoza920) [inline .env.example:4]: 90b7b8a27f92c93e8ea56af02e956a

File: .env.example [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+# Use 'development' to test with real credentials while developing
+# Use 'production' to go live with real users
```
Comment (Mendoza920) [inline .env.example:6]: 2d88dc4511042b186e8f66945a0a69

File: .env.example [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+PLAID_ENV=sandbox
```
Comment (Mendoza920) [inline .env.example:7]: 90b7b8a27f92c93e8ea56af02e956a

File: .env.example [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+# PLAID_COUNTRY_CODES is a comma-separated list of products to use when
+# initializing Link, e.g. PLAID_PRODUCTS=auth,transactions.
```
Comment (Mendoza920) [inline .env.example:9]: What's Orange County code

**Review by Mendoza920 — APPROVED** [review]

**Comment by skylarmb** [comment]
hi @Mendoza920! Looks like you might be a little lost.

If you are an end user of an application that uses Plaid, you can reach our consumer support team through the chat here: https://my.plaid.com/help. You can read more about Plaid here: https://plaid.com/how-it-works-for-consumers/

If you are a developer trying to create an app that uses Plaid, try filing a support ticket with your questions through the developer dashboard (dashboard.plaid.com). You will also need to rotate the API secrets you pasted publicly here before using them in your app. In the meantime I've verified that the secrets pasted here do not have access to anything other than sandbox, so no harm done.

**Comment by Mendoza920** [comment]
I still don't understand though.

**Comment by skylarmb** [comment]
hi @Mendoza920 what don't you understand?

**Comment by skylarmb** [comment]
@raxod502-plaid I addressed your comments

**Review by raxod502-plaid — APPROVED** [review]
Ah yes! Meant to do this earlier ✨

**Comment by skylarmb** [comment]
merged in https://github.com/plaid/quickstart/pull/167

**Pull request closed without merging**